### ISSUE TITLE
Playback to Saved Search

### ISSUE THREAD

**Issue by knobbysideup** [Graylog2/graylog2-server#7956]
With the more flexible saved search pages, would it be possible to optionally tie a dashboard widget’s replay to a specific saved search? That would be incredibly useful in improving workflow. Obviously, I can just load a search manually from the search page, but loading exactly the view I’m looking for directly from a dashboard widget would be great.

**Comment by dennisoelkers** [comment]
Hey @knobbysideup, thanks for the feedback! 

Can you tell me a bit more about how that feature should look like? I have a widget on a dashboard and a saved search. The saved search and the dashboard are unrelated. Assuming I could replay the widget's search to that saved search, how would the result look like?

**Comment by knobbysideup** [comment]
Right.  I'd like a way to establish that relationship and a way to play it back.  Maybe another playback button if the relationship exists, or a context menu.  It would load the saved search exactly like you would from the search page.

For example, I have a security dashboard.  In that dashboard, one of the widgets is a count of http errors from the DMZ.  I also have a saved search that displays a lot of relevant information about http errors that I can use to drill into specific details to address the problem/attack.  I'd like to be able to quickly get to that saved search from the widget by defining a link between the two.  This would not replace the replay for the widget, which is also useful, but would be an additional playback that links directly to a saved search.
